tests/inspect/inspect.go

```diff
@@ -172,5 +176,49 @@ func main() {
 		time.Sleep(time.Duration(globalFlags.Sleep) * time.Second)
 	}
 
+	if globalFlags.Allocate >= 0 {
+		sizeBytes := globalFlags.Allocate * (1024 * 1024)
+		buf := make([]int32, sizeBytes)
+		buf[0] = 0
+
+		fmt.Println("Allocation done")
```

Comment:
Didn't you say this make the OOM kill the program when it goes over the limit and swap is disabled?

Any idea how to have a meaningful test in that situation?